regression/pages/studio/pages_page_studio.py

```diff
@@ -0,0 +1,69 @@
+"""
+Pages page
+"""
+from edxapp_acceptance.pages.common.utils import (
+    wait_for_notification,
+    click_css)
+
+from regression.pages.studio.course_page_studio import CoursePageExtended
+
+
+class PagesPageExtended(CoursePageExtended):
+    """
+    Pages page.
+    """
+    url_path = "tabs"
+
+    def is_browser_on_page(self):
+        return self.q(css='body.view-static-pages').visible
+
+    def add_page(self):
+        """
+        Adds a new empty page.
+        """
+        click_css(self, '.button.new-button.new-tab', 0, False)
+        self.wait_for_element_visibility(
+            '.component.course-tab.is-movable', 'New page is not visible')
```

Comment:
Please format this according to http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#breaking-long-lines